<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Email</class>
 <widget class="QWidget" name="Email">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="welcome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>16</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inboxBT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>接收邮件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outboxBT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发送邮件</string>
   </property>
  </widget>
  <widget class="QLabel" name="senderL">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发件人：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="senderLE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>761</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="receiverLE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>761</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="recevierL">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>收件人：</string>
   </property>
  </widget>
  <widget class="QLabel" name="subjectL">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>主题：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subjectLE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>761</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="contentTE">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>831</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBT">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>510</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBT">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>670</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="attachmentL">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>附件:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanBT">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>510</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QListWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>831</width>
     <height>64</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="receiveList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>66</y>
     <width>831</width>
     <height>500</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>500</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="draftBT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>草稿箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBT">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>670</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>存草稿</string>
   </property>
  </widget>
  <widget class="QListWidget" name="draftList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>831</width>
     <height>500</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>500</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
